### PR TITLE
other small dep update

### PR DESCRIPTION
**Pull request by attila-lin**

### CODE REVIEW

File: poem/src/web/websocket/utils.rs [hunk 1]
```diff
@@ -49,6 +49,7 @@ impl From<tokio_tungstenite::tungstenite::Message> for Message {
             Binary(data) => Message::Binary(data),
             Ping(data) => Message::Ping(data),
             Pong(data) => Message::Pong(data),
+            Frame(_) => unimplemented!("frame is not supported"),
```
Comment (sunli829) [inline poem/src/web/websocket/utils.rs:52]:
```suggestion
            Frame(_) => unreachable!(),
```

Comment (attila-lin) [reply poem/src/web/websocket/utils.rs]: Ok

**Comment by attila-lin** [comment]
changed the test because [link](https://github.com/snapview/tungstenite-rs/blob/4e6d339a4e825a737be5fc878c9675b6238a5fe5/src/handshake/client.rs#L113)

**Comment by sunli829** [comment]
Thanks! 😁

**Pull request merged**